### PR TITLE
Updates/home

### PR DESCRIPTION
**Pull request by andrewJamesBaker**
Polished home screen. There is a lot of extra code that will need to be removed that is unnecessary. Commented out or nested in an if statement. Feel free to play around with all of the placements of the cards and bars, along with my other designs.

### CODE REVIEW

**Review by kny4 — APPROVED** [review]
looks good! I appreciate that you've literally left all the code (even just in comments) and I can see the changes easily. I think maybe the habit stack sentence can just be a sentence, but I know you just copied the code from the Habit stack page, which is totally fine!

**Comment by Bongo9911** [comment]
I made some adjustments to the home page:

- Text at the top bar is now 100% opaque, with the opacity only being reduced on the background now
- Aligned content to the center of their flex boxes
- Made text fit inside input boxes on smaller screens (may need to be adjusted further with some way of adjusting it relative to the screen size)

**Comment by Bongo9911** [comment]
Also, note: don't merge the conflicts with the home screen yet or that will break this branch

File: package-lock.json [hunk 1]
```diff
@@ -1682,7 +1682,7 @@
         "chalk": "^3.0.0",
         "lodash": "^4.17.15",
         "mime": "^2.4.1",
-        "node-fetch": "^2.6.1",
```
Comment (Bongo9911) [inline package-lock.json:1685]: Keep this at 2.61

**Review by Bongo9911 — APPROVED** [review]

**Comment by JoePastucha** [comment]
I think we should potentially change "your habit" to "Goal" to stay constant with the profile page. Or change the profile page to stay constant. Either way, I like it!

**Review by JoePastucha — APPROVED** [review]

**Comment by zeph01** [comment]
> I think we should potentially change "your habit" to "Goal" to stay constant with the profile page. Or change the profile page to stay constant. Either way, I like it!

Or "My Habit" so that it's more personal?

**Comment by zeph01** [comment]
And I think it might look better if the Habit is placed on top of the Habit Stack instead of putting them side by side.

**Review by zeph01 — APPROVED** [review]
Looks great to me! :))
Hm I am not a huge fan of putting a picture for the background. But we can talk about that.

**Pull request merged**